bread/formatters.py

```diff
@@ -59,12 +58,6 @@ def as_text(value):
     return mark_safe(linebreaks(value, autoescape=True))
 
 
-def as_time(value):
-    if value.tzinfo:
-        value = value.astimezone(tz.gettz(settings.TIME_ZONE))
-    return f"{value.hour:02}:{value.minute:02}:{value.second:02}"
-
-
```

Comment:
Unused, has an unnecessary dependency to dateutil, Python stdlib and the dateparser library should do a better job.